### DISCUSSION TITLE
Shouldn't the ReVanced-patches repository be pinned on the ReVanced GitHub page?

### DISCUSSION THREAD

**Discussion by KobeW50** [ReVanced/revanced-discussions#1032] [General]
Title. Currently the ReVanced-patches-temlate repo is pinned and the (new) ReVanced-patches repo is not pinned. It's fine if the template repo is pinned, but the normal patches repo should also be pinned. Maybe remove the patcher repo from the pins?

(I'm bringing this up here bec I'm too lazy to join the discord again.)

**Comment by oSumAtrIX** [comment]
I'm not sure exactly but for me the patches repo is pinned

**Reply by oSumAtrIX** [reply]
Looks like there's a pin for members view and for public view, fixed.

PS: Discord is not our first platform of choice. You can use GitHub discussions just fine.

**Comment by LisoUseInAIKyrios** [comment]
Integrations could also be pinned, since it's required for CLI building.

**Reply by oSumAtrIX** [reply]
Unfortunately there's not enough pin space. We would need to remove another pin. Any suggestions?

**Reply by LisoUseInAIKyrios** [reply]
Maybe remove the website pin.  Seems unlikely someone would frequently visit that repo.
